### DISCUSSION TITLE
Çifte Güzellik

### DISCUSSION THREAD

**Discussion by AcelyaP** [sonsuzus/sonsuzus#327] [Soru Cevap]
Size bir A dizisi verildi, uzunluk N. K'yi seçebilirsiniz farklı dizinler ve içlerindeki öğeyi ikiye katlayın.
Dizinin güzelliği, K uzunluğu ile herhangi bir alt dizinin toplamıdır. A dizisinin tüm permütasyonlarında maksimum güzelliği bulun.

**Giriş Formatı:**

- İlk satır bir tamsayı T içerir, test vakalarının sayısını belirtir.
- Her test senaryosunun ilk satırı iki tam sayı sırasıyla N ve K içerir .
- Her test senaryosunun ikinci satırında N dizinin öğelerini gösteren boşlukla ayrılmış tamsayılar vardır.

**Çıkış biçimi:**
Her test durumu için, A'nın tüm permütasyonlarının üzerine maksimum güzelliği yazdırın.

**Kısıtlamalar:**
1<=T<=10
1<=K<=N<=10⁵
1<=A[i]<=10³

**Comment by sonsuzus** [comment]
Burada anlamadım ne demek istediğini? Sen anladın mı soruyu?

**Reply by AcelyaP** [reply]
hocam ben sorudan çevirince böyle oluyor en düzgün

**Reply by sonsuzus** [reply]
Örnekleri de ekle. Bir de sorunun linkini.
